docs/reference_guides/state_space.rst

```diff
@@ -3,156 +3,187 @@ The State Space
+Bundling states with a similar representation and a symmetric treatment in the
+model solution together avoids double calculation and allows to write simpler
+and more efficient code.
+This consideration constitutes the base for the fine grained division of states
+in respy which is defined in the period_choice_cores.
+The range of methods contained in our state space facilitates clear communication
+between different groups and representations of the state space.
+This guide contains an illustration of the most important components of the state space
+by stressing their role in representation of states, division of states and
+communication between states.
+
+
+Representation
+--------------
+We divide state space dimensions in core and dense dimensions.
+A core dimension is a deterministic function of past choices, time
+and initial conditions.
+Adding another core dimension changes the structure of the state space
+in a complicated way.
+A dense dimension is not such a function and thus either changes the state space
+in a more predictable way or changes the state space in such a complicated way that
+we find it easier to have slightly larger state space than required rather than
+performing complciated calculations to get a precise one.
+A prime example for a dense variable in pur mode is an observable characteristic
+that does not change in the model.
+The addition of exogenous processes as dense variables makes the
+distinction a bit less explicit but the general logic still applies.
+A dense state of the model is a combination of its dense
+and core states.
+All attributes that are only functions of either dense dimensions or
+core dimensions can be calculated seperately and can be combined
+thereafter.
+This property dramatically reduces the amount of calculations required
+to obtain several pieces of information.
+The main conceptual objects underlying the representation of states
+are the core state space, the dense grid and the dense state space.
 
 .. _core_state_space:
 
-The Core State Space
---------------------
-
-ads
+- The Core State Space:
+    The core state space is the set of all core states. It is the image of the function
+    underlying the core dimensions invoked at al feasible combinations of past choices
+    time and initial conditions.
+    The Core State Space is explicitely obtained in the module and is represented by
+    a pd.DataFrame. As such it is the basis for most calculations and every state in the
+    solution is partly represented by apointer to a row in the core state space.
 
-.. _dense_grid:
 
-Dense Grid
-----------
 
-The dense grid is a list that contains all states of dense variables. A dense variable
-is not a deterministic function of past choices and time. Adding another dense variables
-essentially copies the full state space. The disjunction makes use of this property. By
-expressing a state as a combination of a dense and a core state we avoid several
-duplications.
+.. _dense_grid:
```

Comment:
I think that some of the information below should be moved to the attribute section in the docstring of the state space.